python/pyspark/mllib/tree.py

```diff
@@ -24,7 +24,41 @@
 from pyspark.mllib.linalg import _convert_to_vector
 from pyspark.mllib.regression import LabeledPoint
 
-__all__ = ['DecisionTreeModel', 'DecisionTree', 'RandomForestModel', 'RandomForest']
+__all__ = ['TreeEnsembleModel', 'DecisionTreeModel', 'DecisionTree', 'RandomForestModel',
+           'RandomForest', 'GradientBoostedTrees']
+
+
+class TreeEnsembleModel(JavaModelWrapper):
```

Comment:
In Scala, TreeEnsembleModel is private[tree].  I'd add an underscore before the name (and maybe also add some documentation noting that this class is a private API).